packages/headless/src/features/generated-answer/generated-response-format.ts

```diff
@@ -23,6 +23,8 @@ export interface GeneratedResponseFormat {
   answerStyle: GeneratedAnswerStyle;
   /**
    * The content formats that are supported for rendering the answer. If no values are provided, `text/plain` is used.
+   * The answer is generated using the first format supported by the front-end component, from left to right.
+   * The request fails when no matching format is found. It is recommended to include `text/plain` as the last value in the array to act as a fallback format.
```

Comment:
> when no matching format is found

found by what?

Reply:
It is the first match between formats supported by the front-end component, and formats supported by the model.

Reply:
Sorry for the super late reply, @lbergeron !

How about:

> The request fails when none of the specified formats are supported by the front-end component or by the model.